test/test_utils/__init__.py

```diff
@@ -32,11 +32,13 @@
 UBUNTU_18_BASE_DLAMI_US_EAST_1 = "ami-0e3e654c8b1971d8d"
 PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_EAST_1 = "ami-0673bb31cc62485dd"
 PT_GPU_PY3_BENCHMARK_IMAGENET_AMI_US_WEST_2 = "ami-02d9a47bc61a31d43"
+NEURON_UBUNTU_18_BASE_DLAMI_US_WEST_2 = "ami-0b5d270a84e753c18"
```

Comment:
Is the AMI public?

Reply:
Yup, this is the public Ubuntu 18.04 Base DLAMI.

Also, the 90 GB size requirement is for Neuron, yes.